site/docs/master/migrating-to-velero.md

```diff
@@ -4,7 +4,7 @@ As of v0.11.0, Heptio Ark has become Velero. This means the following changes ha
 
 * The `ark` CLI client is now `velero`.
 * The default Kubernetes namespace and ServiceAccount are now named `velero` (formerly `heptio-ark`).
-* The container image name is now `gcr.io/heptio-images/velero` (formerly `gcr.io/heptio-images/ark`).
+* The container image name is now `velero/velero` (formerly `gcr.io/heptio-images/ark`).
```

Comment:
I think we can just delete this page from `master`, as this isn't linked to anywhere and is a pre-1.0 change.

Reply:
I was wondering where this coming from :)